docs/developers/setup.md

```diff
@@ -31,7 +31,7 @@ If everything worked, we should see the line "> Compiled successfully using:" in
 Ganache-CLI was already installed via `yarn` and now we will use it to run a test-chain with this command:
 
```

Comment:
Content changes are good. Thoughts on just moving this out of the `Quick Start` section and making `Setup` its own top level page, since there is only one thing in `Quick Start`?